README.md

```diff
@@ -138,49 +147,107 @@ const config = {
+system will still function correctly and migrate correctly:
 
 ```js
-const sbot = SecretStack({appKey: caps.shs})
+const sbot = SecretStack({ caps })
   .use(require('ssb-db'))
   .use(require('ssb-db2'))
   .use(require('ssb-db2/compat'))
   .call(null, config)
 ```
 
-However, note that while the old log exists, it will be continously migrated to the new log, and ssb-db2 forbids you to use its database-writing APIs such as `add()`, `publish()`, `del()` and so forth, to prevent the two logs from diverging into inconsistent states. The old log will remain the source of truth and keep getting copied into the new log, until the old log file does not exist anymore.
+However, note that while the old log exists, it will be continously
+migrated to the new log, and ssb-db2 forbids you to use its
+database-writing APIs such as `add()`, `publish()`, `del()` and so
+forth, to prevent the two logs from diverging into inconsistent
+states. The old log will remain the source of truth and keep getting
+copied into the new log, until the old log file does not exist
+anymore.
 
 ### Migrating without including ssb-db2
 
-Because ssb-db2 also begins indexing basic metadata once it's included as a plugin, this may cost more (precious) CPU time. **If you are not yet using db2 APIs** but would like to migrate the log anyway, in preparation for later activating db2, then you can include only the migration plugin, like this:
+Because ssb-db2 also begins indexing basic metadata once it's included
+as a plugin, this may cost more (precious) CPU time. **If you are not
+yet using db2 APIs** but would like to migrate the log anyway, in
+preparation for later activating db2, then you can include only the
+migration plugin, like this:
 
 ```js
 const sbot = SecretStack({appKey: caps.shs})
   .use(require('ssb-db2/migrate'))
   .call(null, config)
 ```
 
-Note that the `start` behavior is the same: you can either start it automatically using `config.db2.automigrate` or manually like this:
+Note that the `start` behavior is the same: you can either start it
+automatically using `config.db2.automigrate` or manually like this:
 
 ```js
 sbot.db2migrate.start()
 ```
 
 ## Methods
 
-FIXME: add documentation for these
+### get(msgId, cb)
+
+Get a particular message by message id.
+
+### del(msgId, cb)
+
+Delete a specific message given the message id from the
+database. Please note that this will break replication for anything
+trying to get that message, like createHistoryStream for the author or
+EBT. Because of this, it is not recommended to delete message with
+this method unless you know exactly what you are doing.
+
+### deleteFeed(feedId, cb)
+
+Delete all messages of a specific feedId. Compared to `del` this
+method is safe to use.
+
+### publish(msg, cb)
+
+Convinience method for validating and adding a message to the database
+written by the feed running the secret-stack.
+
+### add(msg, cb)
+
+Validate and add a raw message (without id and timestamp) to the
```

Comment:
Should we say "message value" as in `msg.value` instead? Maybe it helps clarify what this object should look like?